src/metadata/serializeType.ts

```diff
@@ -84,6 +84,36 @@ function serializeTypeReferenceNode(
   );
 }
 
+function serializeTypeImportNode(
+  className: string,
+  node: t.TSImportType
+) {
+  /**
+   * We need to save references to this type since it is going
+   * to be used as a Value (and not just as a Type) here.
+   *
+   * This is resolved in main plugin method, calling
+   * `path.scope.crawl()` which updates the bindings.
+   */
+  const reference = serializeReference(node.qualifier!);
```

Comment:
Thank you for your effort! I'd like to know here when `qualifier` may be null (maybe handling it with a simple `return t.identifier('Object');` in this case): do you have more info about the `TSImportType`?

Reply:
I dont know when the qualifier can be null, I just followed the TypeScript declaration for TSImportType

Reply:
may be when the dynamic import not found